SharpPcap/LibPcap/LibPcapSafeNativeMethods.Interop.cs

```diff
@@ -41,7 +41,14 @@ internal static partial class LibPcapSafeNativeMethods
         //       This file is called $assembly_name.dll.config and is placed in the
         //       same directory as the assembly
         //       See http://www.mono-project.com/Interop_with_Native_Libraries#Library_Names
-        private const string PCAP_DLL = "wpcap";
+        private const string PCAP_DLL = 
+#if NET6_0_OR_GREATER
+            "pcap"
+#else
+            "wpcap"
+#endif
```

Comment:
This will work if you are static linking to Linux, but not if you are static linking to Windows.

Reply:
I don't think we need static linking on Windows because the environment is way easier to setup, and there's no router running on Windows.